refiner/README.md

```diff
@@ -41,6 +41,25 @@ To build the Docker image for the Message Refiner from source instead of downloa
 3. Navigate to the top-level directory of this repository.
 4. Run `docker build -t refiner .`.
 
+## Deploying to Demo
+
+We have a [GitHub action](https://github.com/CDCgov/dibbs-ecr-refiner/actions/workflows/trigger_demo_workflow.yaml) to deploy to our demo environment on demand whenever a new release is ready to go out. **Doing so will cause the demo site to temporarily go down / require a manual step to get it back up, so deploy using this method only when a few seconds of downtime is acceptable.**
+
+### Prerequiste permissions
+
+- Access/permissions to run actions in the Skylight infra repo.
+- Access to the Skylight Azure environment and the resources within the Refiner container instances / DIBBs app hub.
+- Ability to run actions in the Refiner repo.
```

Comment:
We should add here how to get this access. Something generic even would be helpful in case folks don't have the access that's required.